tree/tree/src/TBasket.cxx

```diff
@@ -1284,8 +1301,16 @@ Int_t TBasket::WriteBuffer()
             R__SizeBuffer(*cascade_buffer, buflen);
             cascade_buffer->SetWriteMode();
             precisionCascades.push_back( cascade_buffer->Buffer() + cascade_basket->GetKeylen() );
+            cxlevels.push_back(cxlevel); // WRONG! From where do we get the cxlevels of the precision cascade?
```

Comment:
It comes from the `configArray` and to extract the information (should probably be done within `R__zip....`)

```
if (configArray && configArray->IsPrecisionCascade()) { // just in case something is wrong
    // oups I was not uploaded the implementation yet :(
}
```

Reply:
OK....I'll await your implementation and then I'll write the relevant code in RZip.cxx.

Reply:
See https://github.com/pcanal/root/pull/13/files#r900478838